### PR TITLE
fix(db): fix functions search path for backups

### PR DESCRIPTION
**Pull request by thepiwo**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,7 +5,10 @@ node_modules
 .idea
 .env
 /logs/*.log
+/logs/*.json
 ./images/
 /traces/*.json
 docker-compose.override.yml
 swagger.json
+images/
```
Comment (kenodressel) [inline .gitignore:13]: looks similar to `./images/` above

File: migrations/20210618-fix-function-search-path.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                }
+                next();
+            });
+        }
+        if (this.useTransaction) {
+            return queryInterface.sequelize.transaction(run);
+        } else {
+            return run(null);
+        }
+    },
```
Comment (kenodressel) [inline migrations/20210618-fix-function-search-path.js:54]: omit this?

File: migrations/20210618-fix-function-search-path.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    "created": "2021-06-18T11:11:39.431Z",
+    "comment": ""
+};
+
+var migrationCommands = function(transaction) {
+    return [];
+};
+var rollbackCommands = function(transaction) {
+    return [];
+};
```
Comment (kenodressel) [inline migrations/20210618-fix-function-search-path.js:24]: remove this as well

File: .gitignore (hunk 1 above)
Comment (thepiwo) [reply .gitignore:13]: didn't work for me

File: migrations/20210618-fix-function-search-path.js (hunk 2 above)
Comment (thepiwo) [reply migrations/20210618-fix-function-search-path.js:54]: its default in any, so idk

File: migrations/20210618-fix-function-search-path.js (hunk 3 above)
Comment (thepiwo) [reply migrations/20210618-fix-function-search-path.js:24]: its default in any, so idk

File: migrations/20210618-fix-function-search-path.js (hunk 2 above)
Comment (kenodressel) [reply migrations/20210618-fix-function-search-path.js:54]: it wont break anything and make the file easier to read eg like https://github.com/aeternity/tipping-community-backend/blob/main/migrations/20201005-postfix-tip-id.js

File: .gitignore [hunk 4]
```diff
@@ -5,7 +5,10 @@ node_modules
 .idea
 .env
 /logs/*.log
+/logs/*.json
 ./images/
```
Comment (kenodressel) [inline .gitignore:9]:
```suggestion
```

File: migrations/20210618-fix-function-search-path.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+    up: async function(queryInterface)
+    {
+      const transaction = await queryInterface.sequelize.transaction();
+      await queryInterface.sequelize.query('ALTER FUNCTION unclaimed(numeric, text, varchar) SET search_path=public;', { transaction });
+      await queryInterface.sequelize.query('ALTER FUNCTION unclaimed_amount(numeric, text, varchar, numeric) SET search_path=public;', { transaction });
+      await queryInterface.sequelize.query('ALTER FUNCTION claimed_amount(numeric, text, varchar, numeric) SET search_path=public;', { transaction });
+      return transaction.commit();
+    },
+  down: () => {},
```
Comment (kenodressel) [inline migrations/20210618-fix-function-search-path.js:31]:
```suggestion
    down: () => {},
```

**Review by kenodressel — APPROVED** [review]

**Pull request merged**